bin/minifollowups/pycbc_foreground_minifollowup

```diff
@@ -56,6 +57,13 @@ parser.add_argument('--analysis-category', type=str, required=False,
                     help='Designates whether to look at foreground triggers '
                          'background triggers (including "little dogs") '
                          'or background triggers (with "little dogs" removed)')
+parser.add_argument('--sort-variable', default='stat',
+                    help='Which subgroup of --analysis-category to use for '
+                         'sorting. Default=stat')
+parser.add_argument('--sort-order', default='descending',
```

Comment:
I guess this should then be ascending for FAR? Basically, I want the default values to be appropriate for "standard" runs.

Reply:
FAR isn't stored - IFAR is, so it will remain descending. Default values changed